pkg/pool-weighted/contracts/managed/ManagedPool.sol

```diff
@@ -215,6 +238,21 @@ contract ManagedPool is ManagedPoolSettings {
         // amountsIn are amounts entering the Pool, so we round up.
```

Comment:
One slightly yucky difference between this and `ComposableStablePool` is that `ComposableStablePool` take a dummy entry in the initialisation userData whereas we don't here. This way is "better" but I want to flag up the incompatability.